projects/make-recall-decisions-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/messaging/Handler.kt

```diff
@@ -10,10 +11,22 @@ import uk.gov.justice.digital.hmpps.telemetry.notificationReceived
 @Component
 class Handler(
     override val converter: NotificationConverter<HmppsDomainEvent>,
+    private val recommendationStarted: RecommendationStarted,
     private val telemetryService: TelemetryService
 ) : NotificationHandler<HmppsDomainEvent> {
     override fun handle(notification: Notification<HmppsDomainEvent>) {
         telemetryService.notificationReceived(notification)
-        TODO("Not yet implemented")
+        notification.message.personReference.findCrn()?.let {
```

Comment:
It's unlikely to ever happen, but should we error if the message doesn't have a CRN?